homework24/src/ba/bitcamp/homework24/task1/ComplaintPrototype/Complaint.java

```diff
@@ -0,0 +1,158 @@
+package ba.bitcamp.homework24.task1.ComplaintPrototype;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Prototype of Complaint, integer for primary key, strings for complaint
+ * message and date. Date is setup in HH:MM:SS [DD.MM.YYY] format so full
+ * current date of complaint will be logged.
+ * 
+ * @author boris
+ *
+ */
+public class Complaint {
+
+	private Integer id;
+	private String complaint;
+	private String date;
+
+	/**
+	 * Default constructor of Complaint class, initializes id to null, and sets
+	 * current date.
+	 */
+	public Complaint() {
+		this.id = null;
+		setDate();
+	}
+
+	/**
+	 * Constructor of Complaint class, initializes it to null, sets current date
+	 * and lets user input own complaint message.
+	 * 
+	 * @param complaint
+	 *            <code>String</code> type value of complaint
+	 */
+	public Complaint(String complaint) {
+		this.id = null;
+		this.complaint = complaint;
+		setDate();
+	}
+
+	/**
+	 * Constructor of Complaint class, used to create object when complaint is
+	 * read from database.
+	 * 
+	 * @param id
+	 *            <code>Integer</code> type value of primary key
+	 * @param complaint
+	 *            <code>String</code> type value of complaint message
+	 * @param date
+	 *            <code>String</code> type value of date when complaint was
+	 *            logged
+	 */
+	public Complaint(Integer id, String complaint, String date) {
+		this.id = id;
+		this.complaint = complaint;
+		this.date = date;
+	}
+
+	/**
+	 * Private setter for date, formated to show all information,
+	 */
+	private void setDate() {
```

Comment:
GUI ce formatirati datum kako to korisnik zeli, a ne da ti to radis u backend-u, jer to i jeste svrha GUI-a ;). Npr, sta ako imas pet jezika za izbor i korisnik zeli da ima svoj format datuma za prikaz na osnovu regional postavki?